sdk/tables/Azure.Data.Tables/samples/Sample5UpdateUpsertEntities.md

```diff
@@ -0,0 +1,71 @@
+# Updating and Upserting Table Entities
+
+This sample demonstrates how to query a table for entities. You will have needed to [create a table](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/tables/Azure.Data.Tables/samples/Sample1CreateDeleteTables.md) in the service in order to query entities from it. To get started, you'll need access to either a Storage or Cosmos DB account.
+
+## Create a `TableClient`
+
+A `TableClient` is needed to perform table-level operations like inserting and deleting entities within the table, so it is ideal for dealing with only a specific table. There are two ways to get a `TableClient`:
+- Call `GetTableClient` from the `TableServiceClient` with the table name.
+
+```C# Snippet:TablesSample1GetTableClient
+tableName = "OfficeSupplies1p2";
+var tableClient = serviceClient.GetTableClient(tableName);
+```
+
+- Create a `TableClient` with a SAS URI, an endpoint and `TableSharedKeyCredential`, or a connection string.
+
+```C# Snippet:TablesSample1CreateTableClient
+tableClient = new TableClient(
+    new Uri(storageUri),
+    tableName,
+    new TableSharedKeyCredential(accountName, storageAccountKey));
+```
+
+If you are not familiar with creating tables, refer to the sample on [creating and deleting tables](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/tables/Azure.Data.Tables/samples/Sample1CreateDeleteTables.md).
+
+## Upsert an entity
+
+The upsert operation combines the capabilities of insert and update unconditionally.
```

Comment:
You may want to just touch on the semantics of an upsert for those unfamiliar... or maybe link to it somewhere?

Reply:
I debated it, but the snippet comments touch on them. Do you think it makes sense to expand on it more here also?

Reply:
I had no idea what an upsert was but the text gave me enough context to understand.

Reply:
I'll defer to your judgement;  I'm tainted by knowing what it is and trying to project "what would I think if I didn't...."  so I may be a poor perspective.